_layouts/default.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
this is a really good layout template! i like how you have the `lang` tag!

Reply:
cool 😎